packages/maskbook/src/social-network/PostInfo.ts

```diff
@@ -1,5 +1,5 @@
 import { PostInfo } from '@dimensiondev/mask-plugin-infra'
-export { PostInfo } from '@dimensiondev/mask-plugin-infra'
+export { PostInfo }
```

Comment:
Well, so this is strange... I think they're identical

Reply:
Yep, it looks like https://github.com/babel/babel-loader/issues/603 (but it was [resolved in @babel/preset-typescript@7.12](https://github.com/babel/babel-loader/issues/603#issuecomment-737287260)).

Reply:
but we're not using babel